src/server/src/main/java/io/cassandrareaper/storage/MemoryStorageFacade.java

```diff
@@ -127,4 +179,122 @@ public IClusterDao getClusterDao() {
     return this.memClusterDao;
   }
 
-}
\ No newline at end of file
+  private void persist(Object... objects) {
```

Comment:
**Question:** Shouldn't we make this a `synchronized` method to avoid race conditions?

Reply:
Yes, but I may opt to just do a synchronized block inside the method if I can get it so that reads to the store are allowed concurrently and only writes are synchronized.